File: tree/theory/tree0.cpp
#include <bits/stdc++.h>
using namespace std;


class Node
{
public:
    int data;
    Node* left;
    Node* right;

    Node(int value)
    {
        data = value;
        left = nullptr;
        right = nullptr;
    }
};


int main()
{
    Node* root = new Node(10);
    root->left = new Node(20);
    root->right = new Node(30);
    
    cout<< "Root: "<< root->data << endl;
    cout<< "Left child: "<< root->left->data << endl;
    cout<< "Right child: "<< root->right->data << endl;

return 0;
}

File: tree/tree0.py

class Node:
    def __init__(self, value):
        self.value = value
        self.left = None
        self.right = None

    def __repr__(self):
        return f"Node({self.value})"

def main():
    root = Node(10)
    root.left = Node(20)
    root.right = Node(30)

    print("Root:", root.value)
    print("Left child:", root.left.value)
    print("Right child:", root.right.value)

if __name__ == "__main__":
    main()